### PR TITLE
Update LibreWolf.LibreWolf to v94.0.2.3

### PR DESCRIPTION
**Pull request by maltejur**
This pull-request was auto-generated by [maltejur/librewolf-winget](https://github.com/maltejur/librewolf-winget) and manualy triggered by @stanzabird

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/35358)

### CODE REVIEW

File: manifests/l/LibreWolf/LibreWolf/94.0.2.2/LibreWolf.LibreWolf.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+PackageIdentifier: LibreWolf.LibreWolf
+PackageVersion: 94.0.2.2
```
Comment (OfficialEsco) [inline manifests/l/LibreWolf/LibreWolf/94.0.2.2/LibreWolf.LibreWolf.installer.yaml:2]:
```suggestion
PackageVersion: 94.0.2
```

File: manifests/l/LibreWolf/LibreWolf/94.0.2.2/LibreWolf.LibreWolf.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+PackageIdentifier: LibreWolf.LibreWolf
+PackageVersion: 94.0.2.2
```
Comment (OfficialEsco) [inline manifests/l/LibreWolf/LibreWolf/94.0.2.2/LibreWolf.LibreWolf.locale.en-US.yaml:2]:
```suggestion
PackageVersion: 94.0.2
```

File: manifests/l/LibreWolf/LibreWolf/94.0.2.2/LibreWolf.LibreWolf.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+PackageIdentifier: LibreWolf.LibreWolf
+PackageVersion: 94.0.2.2
```
Comment (OfficialEsco) [inline manifests/l/LibreWolf/LibreWolf/94.0.2.2/LibreWolf.LibreWolf.yaml:2]:
```suggestion
PackageVersion: 94.0.2
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Hmm, you fixed the version, however i can't see anywhere that its 94.0.2.2
image

**Comment by maltejur** [comment]
This seems to be a general issue with this version, even if you install it with the [installer](https://gitlab.com/librewolf-community/browser/windows/-/releases). v94.0.2.2 is different from  v94.0.2, but still installs as v94.0.2. I've opened a issue about that on the librewolf windows repository [here](https://gitlab.com/librewolf-community/browser/windows/-/issues/114).

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> This seems to be a general issue with this version, even if you install it with the installer. v94.0.2.2 is different from v94.0.2, but still installs as v94.0.2. I've opened a issue about that on the librewolf windows repository here.

You can set this PR in draft to prevent it showing up in my filter: `is:pr is:open draft:false -label:Moderator-Approved -label:Project-File -label:Blocking-Issue -label:Needs-Author-Feedback -label:Validation-Metadata -label:Author-Not-Authorized`

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
The version seems to be fixed, other than the `v` you added back 😁

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**